tests/infrastructure/hub-and-spoke-vpns/test_firewall.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"Test firewall resources creation in root module."
+
+
+import pytest
+
+@pytest.fixture(scope='module')
+def firewall_modules(plan):
+  names = ['module.firewall-%s' %
```

Comment:
nit: I would filter for modules with `name.startswith('module.firewall-')` so if we change names the test won't break